### ISSUE TITLE
The questions running on the training task

### ISSUE THREAD

**Issue by wufaxiang** [keras-team/autokeras#211]
Hi, I have some questions as following while I run the tasks:
1. I always get the warning after the training model log in console, Should I care it, or how to fix it? `/home/faxiang/anaconda3/lib/python3.6/multiprocessing/semaphore_tracker.py:143: UserWarning: semaphore_tracker: There appear to be 1 leaked semaphores to clean up at shutdown
  len(cache))`
2. When I set time limit to 10 minutes in `fit function`, but it take more than 10 minites to end. I also found the info `Time is out.` in log console. 
3. I got the this warning before the accuracy line.  What's that mean?
`/home/faxiang/anaconda3/lib/python3.6/site-packages/autokeras/bayesian.py:148: UserWarning: Predicted variances smaller than 0. Setting those variances to 0.
  warnings.warn("Predicted variances smaller than 0. "` 
4. I found that the source code was using the pytorch framework, but it showed `Using TensorFlow backend.` in the log console, haha.
5. I used the example code to run on the cifar10 datasets, but after a one-night training, the accuracy is so lower than expected. The best accuracy I got was just 0.78, I found the best error rate in your paper is 0.123. So how can I get that result?

My code on mnist dataset
```
torch.cuda.empty_cache()
(x_train, y_train), (x_test, y_test) = mnist.load_data()
x_train = x_train.reshape(x_train.shape + (1,))
x_test = x_test.reshape(x_test.shape + (1,))
clf = ImageClassifier(verbose=True, path='outputs', augment=False)
clf.fit(x_train, y_train, time_limit=10 * 60 )
clf.final_fit(x_train, y_train, x_test, y_test, retrain=True)
y = clf.evaluate(x_test, y_test)
print(y * 100)
``` 
My code on cifar10 datasets
```
torch.cuda.empty_cache()
(x_train, y_train), (x_test, y_test) = cifar10.load_data()
clf = ImageClassifier(verbose=True, augment=False)
clf.fit(x_train, y_train, time_limit=12 * 60 * 60)
clf.final_fit(x_train, y_train, x_test, y_test, retrain=True)
y = clf.evaluate(x_test, y_test)
print(y * 100)
```

**Comment by tl-yang** [comment]
I think 1. is related to #200   
2. How much longer did it take?  The time_limit is not 100% accurate. However, it shouldn't be too far away from 10 min.
4.   Pytorch itself uses tensorflow backend. It's a normal behavior I believe.

**Comment by erichhhhho** [comment]
2. I think there is a final_fit after the time_limit

**Comment by vakili73** [comment]
I got the warning 3 which @wufaxiang said. thanks!

OS: Win 10 x64 - v1803
Terminal: git version 2.19.0.windows.1
Python: Anaconda Python 3.6

tensorflow-gpu==1.11.0
autokeras==0.2.15
pytorch==0.4.1

I run the original example code:
```
from keras.datasets import mnist
from autokeras.image_supervised import ImageClassifier

if __name__ == '__main__':
    (x_train, y_train), (x_test, y_test) = mnist.load_data()
    x_train = x_train.reshape(x_train.shape + (1,))
    x_test = x_test.reshape(x_test.shape + (1,))

    clf = ImageClassifier(verbose=True)
    clf.fit(x_train, y_train, time_limit=12 * 60 * 60)
    clf.final_fit(x_train, y_train, x_test, y_test, retrain=True)
    y = clf.evaluate(x_test, y_test)
    print(y)
```

_Problem_: Why?
```
C:\ProgramData\Anaconda3\lib\site-packages\autokeras\bayesian.py:151: UserWarning: Predicted variances smaller than 0. Setting those variances to 0.
  warnings.warn("Predicted variances smaller than 0. "
```
